docs/serving/using-auto-tls.md

```diff
@@ -19,41 +44,31 @@ provisioning:
   - [Knative Serving](../install/).
   - [Istio with SDS, version 1.1 or higher](../install/installing-istio.md#installing-istio-with-SDS-to-secure-the-ingress-gateway) or [Gloo, version 0.18.16 or higher](../install/Knative-with-Gloo.md).
     Note: Currently, [Ambassador](https://github.com/datawire/ambassador) is unsupported.
-  - [cert-manager version `0.6.1` or higher](./installing-cert-manager.md).
+  - [cert-manager version `0.12.0` or higher](./installing-cert-manager.md).
 - Your Knative cluster must be configured to use a
   [custom domain](./using-a-custom-domain.md).
 - Your DNS provider must be setup and configured to your domain.
+- If you want to use HTTP01 challenge, you need to configure your custom 
+domain to map to the IP of ingress.
 
 ## Enabling automatic certificate provisioning
 
 To enable support for automatic TLS certificate provisioning in Knative:
 
-1. Determine if `networking-certmanager` is already installed by running the 
-    following command:
-
-    ```shell
-    kubectl get deployment networking-certmanager -n knative-serving
-    ```
-
-1. If `networking-certmanager` is not found, run the following command:
-
-    ```shell
-    kubectl apply --filename https://github.com/knative/serving/releases/download/{{< version >}}/serving-cert-manager.yaml
-    ```
-
-1. Create and add the `ClusterIssuer` configuration file to your Knative cluster
-   to define who issues the TLS certificates, how requests are validated
-   (`DNS-01`), and which DNS provider validates those requests.
+### Step 1: Create cert-manager ClusterIssuer
 
-   1. Create the `ClusterIssuer` file:
+Create and add the `ClusterIssuer` configuration file to your Knative cluster
+to define who issues the TLS certificates, how requests are validated,
+and which DNS provider validates those requests.
 
-      Use the cert-manager reference to determine how to configure your
-      `ClusterIssuer` file:
+1.  ClusterIssuer for DNS01 challenge
```

Comment:
This step doesn't seem clear to me - " ClusterIssuer for DNS01 challenge" seems more like a title than a 1. step.